Cloud Enterprise/Getting Started Examples/gcp/terraform/servers.tf

```diff
@@ -1,26 +1,62 @@
 resource "google_compute_instance" "server" {
-  count = "${length(var.zones)}"
+  count = length(var.zones)
 
   name         = "${var.name}-${element(var.zones, count.index)}"
-  machine_type = "${var.machine_type}"
+  machine_type = var.machine_type
   zone         = "${var.region}-${element(var.zones, count.index)}"
 
   boot_disk {
     initialize_params {
-      size  = 100
+      size  = 30
       image = "projects/ubuntu-os-cloud/global/images/family/ubuntu-1604-lts"
     }
   }
-
+  attached_disk {
+    source = google_compute_disk.disk[count.index].self_link
+  }
   network_interface {
-    subnetwork    = "${google_compute_subnetwork.main.name}"
-    access_config = {}
+    subnetwork    = google_compute_subnetwork.main.name
+    access_config {
+
+    }
   }
 
-  metadata {
+  metadata = {
     managed-by = "terraform"
     ssh-keys   = "${var.remote_user}:${file(var.public_key)}"
   }
+}
+
+resource google_compute_disk "disk" {
```

Comment:
This run okay in TF v0.12 (ditto line 26 above)? It's using the old variable substitution syntax which I think is either deprecated or no longer supported.

Reply:
Yep, when manipulating data, joining vars etc, the "${}" syntax still holds (which kinda makes sense)